### PR TITLE
changed package import name from cpx to cp

### PR DESCRIPTION
**Pull request by andreamah**
# Description:

Allowed for both imports as:
`from adafruit_circuitplayground import cp`
and
`from adafruit_circuitplayground.express import cpx`

This was requested since the first import statement above is how most tutorials present the import statement. 

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Limitations:

# Testing:

- [ ] Test with simulation and deploy to board

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by andreamah** [comment]
It seems that CircuitPython's standard import statement changed a few months ago. It now can automatically detect the CP board type (Bluefruit vs Express). See: https://github.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/commit/cfa005354f1cb587a90fed64b098b5d63cd33498#diff-83e362785431178a0efc45ce23e99323

**Comment by vandyliu** [comment]
Nice fix and nice find!

**Review by vandyliu — APPROVED** [review]

File: src/adafruit_circuitplayground/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .express import cpx as cp
```
Comment (isadorasophia) [inline src/adafruit_circuitplayground/__init__.py:4]: Add a comment [pointing to their PR](https://github.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/pull/79)?

**Review by isadorasophia — APPROVED** [review]
That was simple enough 😮 Cool!

File: src/adafruit_circuitplayground/__init__.py (hunk 1 above)
Comment (andreamah) [reply src/adafruit_circuitplayground/__init__.py:4]: Added!

**Review by markAtMicrosoft — APPROVED** [review]

**Pull request merged**